api/v1alpha1/nnf_access_types.go

```diff
@@ -66,8 +67,7 @@ type NnfAccessStatus struct {
 	// Ready signifies whether status.state has been achieved
 	Ready bool `json:"ready"`
 
-	// Error message
-	Message string `json:"message,omitempty"`
+	Error *dwsv1alpha1.ResourceError `json:"error,omitempty"`
```

Comment:
Is a struct considered "empty" if all its members are empty? Wondering if these need to be pointers

Reply:
Not in the "omitempty" sense. If there are fields in the struct that aren't omitempty, then the struct should be a pointer if you want the whole thing to be omitempty.

So in this case, if there is an error, then I want some of the fields in the ResourceError to be required. If there isn't an error, then the whole ResourceError struct should be omitted.

Reply:
OK - wasn't quite sure